### PR TITLE
added text to gallery and regina five. Also fixed classes and images.

### PR DESCRIPTION
**Pull request by DarkerLonging**

### CODE REVIEW

File: www/artists/afyfe/gallery.html [hunk 1]
```diff
@@ -24,24 +24,27 @@
...
                 </p>
                 <p>
-                    <a href="prarielily.html"><img src="images/prarielily.jpg" alt="Prarie Lily" class='b1'></a> <a href="reflectionnebula.html"><img src="images/reflectionnebula.jpg" alt="The Reflection Nebula" class='b1'></a>
+                    <a href="prarielily.html"><img src="images/prarielily.jpg" alt="Prarie Lily" class="b1"></a> <a href="reflectionnebula.html"><img src="images/reflectionnebula.jpg" alt="The Reflection Nebula" class="b1"></a>
                     <br>
                     Prarie Lily | The Reflection Nebula
                 </p>
                 <p>
-                    <a href="rfiveproject1.html"><img src="images/afyfesinkpainting.jpg" width="500" height="375" alt="no alt text"></a>
+                    <a href="rfiveproject1.html"><img src="images/afyfesinkpainting.JPG" width="500" height="375" alt="no alt text"></a>
```
Comment (gemohandy) [inline www/artists/afyfe/gallery.html]: Why did you change all of the .jpg to .JPG? We specifically changed them the other way because we renamed all of the photos because we don't want capitals in our filenames. Is your local branch malfunctioning?

File: www/artists/afyfe/gallery.html [hunk 2]
```diff
@@ -24,24 +24,27 @@
...
                     <br>
-                    Tribute to the Regina 5
+                    (Summarizes the tribute to fyfe's grandfathers generation of artists. Is she cleaning them up? Or wiping them down the drain?) Tribute to the Regina 5
```
Comment (gemohandy) [inline www/artists/afyfe/gallery.html]: ...You just added what Wayne told you to in the brackets? I'm pretty sure he was asking you to summarize the tribute, not just say "summarizes the tribute".

File: www/artists/afyfe/gallery.html [hunk 3]
```diff
@@ -24,24 +24,27 @@
...
+                </p>
+                <p>
+                    "People always say you suffer for your artwork. ...I didn't know I was going to suffer this much," she said laughing.
```
Comment (gemohandy) [inline www/artists/afyfe/gallery.html]: Did you get this from the Regina Leader Post? It feels a bit awkward for a website, particularly in context. The lines before give no indication of who "she" is, and nothing else is written in this format at all. It should probably just be a quote - remove the bit at the end, and make sure to add an attribution to Amber Fyfe with her name.

File: www/artists/afyfe/index.html [hunk 4]
```diff
@@ -24,7 +24,7 @@
             </div>
             <div id="footer">
                 <p>
-                    Copyright 2008 Amber Fyfe. All images are the property of Amber Fyfe.
+                    "It's not the same feel," said Fyfe. "You only have what the other artist did to look at; you don't have his inspiration or what inspired him."                    <br>
```
Comment (gemohandy) [inline www/artists/afyfe/index.html]: Same sort of thing here - it feels out of place on a website. Don't just blindly copy/paste - reformat it to fit the website, and add proper attribution at the end

File: www/artists/afyfe/rfiveproject1.html [hunk 5]
```diff
@@ -29,33 +29,35 @@
...
                 <br>
-                <img src="images/sink5.jpg" width="200" height="175" alt="Sink 5" style='border-width:1px;'>
-                <br>
-                <a href="http://saskartists.ca/dmorton/"><strong style='text-align:left;'>Doug Morton Sink</strong></a>
+                <img class = "r5img" src="images/sink5.JPG" width="200" height="175" alt="Sink 5" >
                 <br>
+                <a class = "r5" href="http://saskartists.ca/dmorton/">(Colour Purity) Doug Morton Sink</a>
                 <br>
+                <p>
+                    "Everything I tried the way they did it, it went down the drain," said Fyfe.
```
Comment (gemohandy) [inline www/artists/afyfe/rfiveproject1.html]: Same thing as I've mentioned twice now - reformat it for a proper website quote, and add proper attribution.

File: www/artists/afyfe/gallery.html (hunk 1 above)
Comment (DarkerLonging) [reply www/artists/afyfe/gallery.html]: Possibly, i can fix that later. The text i added from the internet and the text before and after images are exactly what wayne told me to put. How would you like me to fix that?

File: www/artists/afyfe/gallery.html (hunk 2 above)
Comment (DarkerLonging) [reply www/artists/afyfe/gallery.html]: Im not sure but i thought he was meaning that the image summarizes the tribute.

File: www/artists/afyfe/gallery.html (hunk 3 above)
Comment (DarkerLonging) [reply www/artists/afyfe/gallery.html]: Yes i did, wayne wanted a fitting quote from the leaderpost.

File: www/artists/afyfe/gallery.html (hunk 2 above)
Comment (gemohandy) [reply www/artists/afyfe/gallery.html]: You can check with him, but I find it unlikely. The image in question is a link to the gallery with all of the other sinks. Also, why else would he have questions in there at the end?

File: www/artists/afyfe/gallery.html (hunk 3 above)
Comment (gemohandy) [reply www/artists/afyfe/gallery.html]: Given that these quotes are just above or in the footer, proper formatting would probably be to make the quote into a single block of text (so things like "said Fyfe" don't interrupt it), and put the name of the source (in this case, Amber Fyfe) after the quote with a marker of some sort between the quote and the name.
Example:
I was born on the planet Earth - Wayne Tunison
